diff --git a/azure-storage-fuse/PKGBUILD b/azure-storage-fuse/PKGBUILD
index 5213ebd..5a567d7 100644
--- a/azure-storage-fuse/PKGBUILD
+++ b/azure-storage-fuse/PKGBUILD
@@ -14,10 +14,18 @@ depends=('boost-libs'
          'libgcrypt')
 makedepends=('boost'
              'cmake')
-source=("$pkgname-$pkgver.tar.gz::https://github.com/Azure/azure-storage-fuse/archive/refs/tags/blobfuse-$pkgver.tar.gz")
-sha256sums=('782e571561b36f706ec36d474db42161318d8193d9deece023b529c13c9e5395')
+source=(
+	"$pkgname-$pkgver.tar.gz::https://github.com/Azure/azure-storage-fuse/archive/refs/tags/blobfuse-$pkgver.tar.gz"
+	"remove-werror.patch"
+)
+sha256sums=(
+	'782e571561b36f706ec36d474db42161318d8193d9deece023b529c13c9e5395'
+	'SKIP'
+)
 
 build() {
+  patch -Np1 -i "${srcdir}/remove-werror.patch"
+
   cd "$pkgname-blobfuse-$pkgver"
 
   # Build Azure Storage lib first
diff --git b/azure-storage-fuse/remove-werror.patch b/azure-storage-fuse/remove-werror.patch
new file mode 100644
index 0000000..3fd71f6
--- /dev/null
+++ b/azure-storage-fuse/remove-werror.patch
@@ -0,0 +1,11 @@
+--- src/azure-storage-fuse-blobfuse-1.4.3/CMakeLists.txt	2022-01-11 11:44:07.000000000 +0800
++++ CMakeLists.txt	2022-03-06 22:31:03.027021868 +0800
+@@ -89,7 +89,7 @@
+   add_definitions(-std=c++11)
+   add_definitions(-D_FILE_OFFSET_BITS=64)
+  # set(WARNING "-Wall -Wextra -Werror -pedantic -pedantic-errors")
+-  set(WARNING "-Wall -Wextra -Werror")
++  set(WARNING "-Wall -Wextra")
+ 
+   set(CMAKE_CXX_FLAGS "${CMAKE_THREAD_LIBS_INIT} ${WARNING} ${CMAKE_CXX_FLAGS}")
+ 
