diff --git a/pmdk/PKGBUILD b/pmdk/PKGBUILD
index 3b42e76..5f76158 100644
--- a/pmdk/PKGBUILD
+++ b/pmdk/PKGBUILD
@@ -5,7 +5,7 @@ pkgname=pmdk
 pkgver=1.11.1
 pkgrel=1
 pkgdesc="Persistent Memory Development Kit"
-arch=('x86_64')
+arch=('x86_64' 'aarch64')
 url="https://pmem.io/"
 license=('BSD')
 depends=(
@@ -13,6 +13,7 @@ depends=(
 )
 makedepends=(
   'pandoc'
+  'time'
 )
 # checkdepends=()
 optdepends=(
@@ -65,6 +66,11 @@ package() {
   # pkgdir=/home/darren/.cache/yay/pmdk/pkg
   make LIB_PREFIX=lib prefix=/usr PREFIX=/usr DESTDIR="$pkgdir/" install
 
+  if [[ -e "$pkgdir/usr/x86_64-linux-gnu/" ]]; then
+    mv -v "$pkgdir/usr/x86_64-linux-gnu/"* "$pkgdir/usr/"
+    rmdir "$pkgdir/usr/x86_64-linux-gnu/"
+  fi
+
   # Fix 01
   # mv -v "$pkgdir/usr/lib64" "$pkgdir/usr/lib"
 
