diff --git a/PKGBUILD b/PKGBUILD
index 0c3b722..09ef524 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -11,7 +11,10 @@ license=("custom:Cling Release License")
 provides=("cling")
 conflicts=("cling")
 depends=("libxml2")
-makedepends=("git" "cmake" "python2-dev" "libxml2-dev")
+focal_depends=("${depends[@]}" "libffi7")
+jammy_depends=("${depends[@]}" "libffi8")
+bullseye_depends=("${depends[@]}" "libffi7")
+makedepends=("git" "cmake" "python2-dev" "libxml2-dev" "libffi-dev" "pkg-config")
 optdepends=(
     "python2: support for scan-view and Jupyter"
     "perl: support for scan-build, ccc-analyzer and c++-analyzer"
